site/content/docs/user/known-issues.md

```diff
@@ -17,12 +17,39 @@ It may additionally be helpful to:
 - reach out and ask for help in [#kind] on the [kubernetes slack]
 
 ## Contents
+* [Failures involving mismatched kubectl versions](#failures-involving-mismatched-kubectl-version)
 * [Docker on Btrfs](#docker-on-btrfs)
 * [Docker installed with Snap](#docker-installed-with-snap)
 * [Failing to apply overlay network](#failing-to-apply-overlay-network)
 * [Failure to build node image](#failure-to-build-node-image)
 * [Failure for cluster to properly start](#failure-for-cluster-to-properly-start)
 
+## Failures involving mistmatched kubectl versions
+
+```bash
+kubectl edit deploy -n kube-system kubernetes-dashboard
+error: SchemaError(io.k8s.api.autoscaling.v2beta1.ExternalMetricStatus): invalid object doesn't have additional properties
+```
+
+You can check your client and server versions by running:
+```bash
+kubectl version
+```
+
+As a workaround, you can install kubectl via homebrew by running:
```

Comment:
I think we should do this only if there is a mismatch between the client and server versions, thus we should explain it

Reply:
im thinking
```
If the client and server versions returned are different, then the cause of the error mention previously, is due to a mismatch between the versions of kubernetes running on the client and server.
```
It's a really long sentence though, do you have any suggestions on breaking it down?

Reply:
What about : if there is a mismatch between the server and client versions, you should install a newer client version. If you are using Mac you can install kubectl via homebrew ...